### DISCUSSION TITLE
🖼 Fanart Competition #3

### DISCUSSION THREAD

**Discussion by BirbSunflower** [sunflower-land/sunflower-land#2827] [Art]
For this competition, we encourage a wide range of art styles and techniques, not limited to pixel art. Feel free to explore your creativity and express yourself in any art style you choose!

Before submitting your artwork, we highly recommend that you read our fanart guidelines, which can be found here:

https://docs.sunflower-land.com/contributing/competitions/fan-art-competition

## Theme
The theme for this month is: **Pumpkin Plaza** ![image](https://github.com/sunflower-land/sunflower-land/assets/123522693/92729cca-e215-473a-9608-73617b7c2ba4)

![image](https://github.com/sunflower-land/sunflower-land/assets/123522693/544896c2-f7b1-4e20-a314-c29c8f3c0313)

Show us your creativity by designing something related to the chosen theme! Whether it's a character that you love, a scene that captivates you, or a moment that leaves a lasting impression, we want to see what you can come up with. Let your imagination soar and share your unique perspective with us! ❤

## Requirements ✏️

- AI-generated art will not be accepted.
- The art should be original and not plagiarised.
- By submitting art, you acknowledge you are the original creator of the art and you give Thought Farm and the community the rights to use it freely in the game and marketing purposes indefinitely.
- You agree to the [terms and services](https://docs.sunflower-land.com/contributing/competitions/fan-art-competition#terms-and-conditions) of the game.

When submitting art, please include the following:

- Your Farm ID.
- Your Discord ID.
- A image of your art with good quality.
- A NPC that represents you. 
- A Google Drive link with proof of your creation. 
To verify the authenticity of your artwork, we kindly request that you provide proof of your creative process. This may include work-in-progress images, concept sketches, time-lapse videos, and, in the case of digital art, we also request the original files.

Please note that the inclusion of NPCs will not have any impact on the judging process of the competition. However, if you choose to provide an NPC, it will be displayed in-game alongside your art piece. If you do not include an NPC, a random one will be selected. Also, a Bumpkin version of your NPC would be appreciated, but not required. 

You can use this image to showcase your NPC:

![image](https://github.com/sunflower-land/sunflower-land/assets/123522693/81132ce6-4863-4be7-a3ca-b5f78434ee60)

## Rewards 🏆
The winner will receive 1000 SFL and a runner-up will receive 500 SFL, however not only will the winning artwork be showcased on the official Sunflower Land Game social media channels and in-game, but it will also be seen by our vibrant community of players from around the world.

We also want to emphasize that even if you don't win, your design may still have the chance to be selected in the future. Our team will carefully review all submissions and may reach out to designers whose work stands out for future collaboration opportunities. Sunflower Land is a community game and we love showcasing and including artwork from our players. Now is your chance to have your artwork immortalised in the #1 Community Web3 game.❤

## Deadline 📅
Artists have until September 4th to submit their entries. After that, our art team will carefully review and analyze each submission to select the winners.

The announcement of the winners will take place during the first week of September.

So, what are you waiting for? Grab your brushes, pens, or digital tools and get creating! Show us what you're made of and join our community of talented artists in celebrating the vibrant world of Sunflower Land Game. We can't wait to see what you come up with!

Good luck to all participants! 🎉

Last Fanart Competition: #2638

**Comment by Vergelsxtn** [comment]
Farm ID: **63**
Discord ID: **Vergelsxtn**
### Meeting friends at Pumpkin Plaza
![Pumpkin-Plaza-3rdperson](https://github.com/sunflower-land/sunflower-land/assets/99605039/6315bc98-1743-407e-9696-4f59c1db5e43)

WIP images here:
https://drive.google.com/drive/folders/1_-1G7Cxv6ZpKWgaGDXBJtfkth---WH9o?usp=sharing

**Comment by Netherzapdos** [comment]
### **Luna**
---
**Farm ID:** 591
**Discord ID:** netherzapdos

Submission:
![Luna Large](https://github.com/sunflower-land/sunflower-land/assets/101855553/a472431c-ba61-4e4c-846e-eab448691e40)

Google Drive with WIPs & Aseprite File:
https://drive.google.com/drive/folders/177TW-WWAUn1INihbvj1tYWkNqVvxVKCJ?usp=sharing

**Comment by holyshit89** [comment]
ID de Granja: #159682
 ID de Discord: holyshit89

FIESTA DE DISFRACES EN LA PLAZA PUMPKIN
![Concurso1](https://github.com/sunflower-land/sunflower-land/assets/141026573/8b651e3d-923d-4640-9de4-83d0b0a0cd58)



Archivos Wip:
https://drive.google.com/drive/folders/1-6Lyt3UxSjwT3rrHIfAyrRT_cMcGPkJo

**Comment by Asterion1989** [comment]
Farm: 55626
Discord: jeansb11

![bud-leaf2](https://github.com/sunflower-land/sunflower-land/assets/78221179/79099e79-0e50-4356-a45e-cfa4976385fd)


[WIP](https://drive.google.com/drive/folders/1wLEjwYGMiJNIACUJMMoUaqw6jgrYvWKD?usp=sharing)
